specification/billing/resource-manager/Microsoft.Billing/stable/2020-05-01/billing.json

```diff
@@ -4704,7 +4719,8 @@
           "enum": [
             "Due",
             "OverDue",
-            "Paid"
+            "Paid",
+            "Void"
```

Comment:
may I ask adding an extra enum "Void" here is by design?

Reply:
Yes, we had missed this earlier.